addons/udes_sale_stock/models/courier.py

```diff
@@ -0,0 +1,7 @@
+from odoo import models, fields
+
+class UdesCourier(models.Model):
+    _name = "udes.courier"
+    _description = "Courier"
+
+    name = fields.Char(string="Courier Name")
```

Comment:
Q: Do we want a view or menuitem to manage these records?

Reply:
Good question

Reply:
Resolving, menuitems and views will be done later